symbiflow-toolchain/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
+
+wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/15/20200608-212841/symbiflow-arch-defs-install-32defbb3.tar.xz | tar -xJ
+
+BIN_DIR=${PREFIX}/bin
+SHARE_DIR=${PREFIX}/share/symbiflow
+
+mkdir -p $SHARE_DIR
+
+cp -r install/share/scripts $SHARE_DIR
+cp -r install/share/techmaps $SHARE_DIR
+cp -r install/bin/* $BIN_DIR/
```

Comment:
What is being installed in `$BIN_DIR/`?

Reply:
The bash scripts to run VPR and other symbiflow-related stuff. 

There is a problem though, that regards some additional python scripts that need to be ported as well (`place_constraints.py`, `io_place.py`, etc). These are needed during the flow, but installing them in this way may generate conflicts with the python conda.

The only solution I thought of for now is to extract those scripts and get them into yet another separate repository (e.g. symbiflow-python-library) and install that throug conda pip, and get rid of the python library within the bin folder.